File: division.py
#Python Program to Read Two Numbers and Print Their Quotient and Remainder

divisor=int(input("enter the divisor value"))
dividend=int(input("enter the dividend value"))

quotient = dividend // divisor

remainder = dividend % divisor

print("quotient is :{0}, remainder is :{1}". 
      format(quotient,remainder)) 

File: division.c
#include<stdio.h>

void main()
{
	int divisor,dividend,quotient,remainder;
	printf("Enter the divisor and dividend  value");
	scanf("%d %d",&divisor,&dividend);
	quotient = dividend/divisor;
	remainder = dividend%divisor;
	printf("the quotient is %d \n remainder is %d \n",quotient,remainder);
}
